src/main/kotlin/eu/europa/ec/eudi/sdjwt/Specs.kt

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2023 European Commission
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package eu.europa.ec.eudi.sdjwt
+
+/**
+ * [Selective Disclosure for JWTs (SD-JWT)](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/)
+ */
+@Suppress("UNUSED")
+object SdJwtSpec {
+    /**
+     * Digests of Disclosures for object properties
+     */
+    const val CLAIM_SD: String = "_sd"
+
+    /**
+     *  Hash algorithm used to generate Disclosure digests and digest over presentation
+     */
+    const val CLAIM_SD_ALG: String = "_sd_alg"
+
+    /**
+     * Digest of the SD-JWT to which the KB-JWT is tied
+     */
+    const val CLAIM_SD_HASH: String = "sd_hash"
+
+    /**
+     * Digest of the Disclosure for an array element
+     */
+    const val CLAIM_THREE_DOTS: String = "..."
+
+    //
+    //  Header parameters, for JWS JSON
+    //
+    /**
+     * An array of strings where each element is an individual Disclosure
+     */
+    const val JWS_JSON_DISCLOSURES = "disclosures"
+
+    /**
+     * Present only in an SD-JWT+KB, the Key Binding JWT
+     */
+    const val JWS_JSON_KB_JWT = "kb_jwt"
+
+    //
+    // Other
+    //
+    const val TILDE: Char = '~'
```

Comment:
@babisRoutis I would call this `DISCLOSURE_SEPARATOR` instead.

Reply:
Yes